specs/src/specs/params.md

```diff
@@ -0,0 +1,50 @@
+# Celestia Governance Params
+
+These are the parameters for mainnet. Note that not all of these parameters are
+changable via governance. This list also includes parameter that require a
+hardfork to change due to being manually hardcoded in the application or they
+are blocked by the `x/paramfilter` module.
+
+## Parameters
+
+| Module.Parameter | Default | Summary | Changeable via Governance |
+| --- | --- | --- | --- |
+| auth.MaxMemoCharacters | 256 | Largest allowed size for a memo in bytes. | True |
+| auth.TxSigLimit | 7 | Max number of signatures allowed in a multisig transaction. | True |
+| auth.TxSizeCostPerByte | 10 | Gas used per transaction byte. | True |
+| auth.SigVerifyCostED25519 | 590 | Gas used to verify Ed25519 signature. | True |
+| auth.SigVerifyCostSecp256k1 | 1000 | Gas used to verify secp256k1 signature. | True |
+| bank.SendEnabled | true | Allow transfers. | False |
+| blob.GasPerBlobByte | 8 | Gas used per blob byte. | True |
+| MaxBlockBytes | 100MiB | Hardcoded value in CometBFT for the protobuf encoded block. | False |
+| MaxSquareSize | 128 | Hardcoded value in the applications that requires a hardfork to change. | False |
```

Comment:
Can you add something like (128*128*512bytes=8MiB) or would this be too verbose?

Reply:
IMO not needed since `MiB` is unambiguous.

Reply:
for square size? atm I just updated to be rows and cols of shares, which I think is more descriptive. We have other descriptions of shares in the `shares.md` and how we encode them which are required to come up with an accurate value